README.md

```diff
@@ -61,12 +61,13 @@ Run the following script in your browser console:
 ```js
 main({
   email: "jordyn@example.com",
-  // Client needs to pass in dynamicLinkSettings, they change depending on
-  // the env and are used by Firebase to generate a sign-in link
+  // These settings are used by Firebase to generate a dynamic sign-in link
   dynamicLinkSettings: {
-    domain: "pasapesos.page.link", // dynamic link domain
-    path: "auth-email", // dynamic link path
-    referrerId: "io.example.app", // app the dynamic link will send you to
+    dynamicLinkDomain: "pasapesos.page.link",
+    url: "https://pasapesos.page.link/auth-email",
```

Comment:
As these are generic examples of how to use recoverysigner-firebase-auth that can be used with any recoverysigner deployment I think we should remove the pasapeso references.

Reply:
fixed!